ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -197,3 +197,17 @@ def logistic(loc=0.0, scale=1.0, size=None):
     u = ivy.random_uniform(low=0.0, high=0.0, shape=size, dtype="float64")
     x = loc + scale * ivy.log(u / (1 - u))
     return x
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def laplace(loc=0.0, scale=1.0, size=None):
+    u = ivy.random_uniform(low=0.0, high=0.0, shape=size, dtype="float64")
+
+    if u < 0.5:
```

Comment:
using an if statement like this doesn't make sense on arrays with multiple values. just looking at the probability density function in the docs (https://numpy.org/doc/stable/reference/random/generated/numpy.random.laplace.html) I don't think you need to include any conditionals like this? please can you try to fix this and make sure your test passes (which you can run with pytest ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py::test_numpy_laplace) 🙂

Reply:
Hey @Sam-Armstrong, 

Thanks for your suggestion :)

I have updated the function. The tests were successful.